### ISSUE TITLE
Can't access /data

### ISSUE THREAD

**Issue by DataDrug** [dtag-dev-sec/t-pot-autoinstall#18]
Basically I dunno what else to try. After I log in with tsec, on ssh or ftp I can't access to /data folder due to lack of premissions. Is there any password for root login somewhere or something ? Sorry for what may be a stupid question, it's late here and all that I wanted to do was take a look at the malware and it's binaries.

**Comment by t3chn0m4g3** [comment]
```
sudo su -
cd /data
```

**Comment by DataDrug** [comment]
I've always had terminal failing with that command. Did a fresh install and works ok.

Is there any way I can submit data found to my virustotal account ?

thanks.

**Comment by t3chn0m4g3** [comment]
No, not all honeypots support it and we opted against it since most users and researchers do prefer not to.
